### DISCUSSION TITLE
ESP32 with Tasmota 10.1.0 won´t set rssi to ZERO when BLE device is gone

### DISCUSSION THREAD

**Discussion by Christian72D** [arendst/Tasmota#14149] [Q&A] [answered]
I have a wroom32 with Tasmota 10.1.0 Bluetooth.
BLE is enabled, the GUI shows my Gigaset beacons (and all other deviced).

The standard behaviour when a device is gone should be this:

    Every beacon report is published as an MQTT tele/%topic%/SENSOR in a separate message:
    If the beacon can no longer be found during a scan and the timeout interval has passed the beacon's RSSI is set to zero (0) and it is no longer displayed in the webUI

[https://tasmota.github.io/docs/Bluetooth/#features_1]

But for me, i get no further message about a device that is out of range.

And i wanted to use this feature for a presence detection.

Any settings i could change?

**Comment by ascillato2** [comment, marked as answer]
Since you haven't had here any answer yet, have you tried in the Discord [Chat](https://discord.gg/Ks2Kzd4)?

**Reply by Christian72D** [reply]
No
